docs/SET_UP.md

```diff
@@ -0,0 +1,199 @@
+# Software Set Up
+This documentation includes the relevant information on how to set up the software frome scratch.
+
+The following software has been tested on Raspberry Pi OS (32-bit) bookworm with Python 3.11.2 and pip 23.0.1 
+
+
+## Raspberry Pi OS
+The Raspberry Pi OS can be installed on a usb stick or SD card by using the Raspberry Pi Imager as described [here](https://www.raspberrypi.com/software/).
+
+We recomend using Raspberry Pi OS (32-bit) bookworm as it the most recent version we have tested on.
+
+When switching on the Raspberry Pi for the first time there will be a short setup stage. For QUB students use the same password used for the other Raspberry Pi
+(if unsure ask a team member), for continuity across all our devices.
+
+## Raspberry Pi Configuration
+To be able to access some of the GPIO pins properly the Pi must be configured.
+
+First of all, if you haven't already connect the Raspberry pi to the internet.
+
+For QUB students see the image below. (This could take a few attempts)
```

Comment:
Why should it take a few attempts?

Reply:
I'm not sure, when I tired to connect it the first time I had to put my password in a few times but once it successfully connects for the first time there are no issues after that